sirepo/package_data/static/js/hellweg.js

```diff
@@ -421,6 +428,15 @@ SIREPO.app.directive('summaryTable', function() {
                 });
             }
 
+            function energySpectrumFromPercentage(value, meanEnergy) {
+                return String(
+                    appState.formatFloat(
+                        ((Number(value.replace(/ %/, '')) * meanEnergy) / 100) * 1000,
```

Comment:
I think this can be more permissive in the regex: `value.replace(/\s%/, '')` in case the amount of whitespace varies